parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -32,10 +32,33 @@ else
 	COCKPIT_VERSION="latest"
 fi
 
-# TODO: with WALinuxAgent>=v2.2.21 (https://github.com/Azure/WALinuxAgent/pull/1005)
-# we should be able to append context=system_u:object_r:container_var_lib_t:s0
-# to ResourceDisk.MountOptions in /etc/waagent.conf and remove this stanza.
+if grep -q ^ResourceDisk.Filesystem=xfs /etc/waagent.conf; then
```

Comment:
think this is worthwhile to try to work around on the grounds that publishing a new image will take days

Reply:
this codepath goes away eventually